ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly-4.14-upgrade-from-aro-4.14.yaml

```diff
@@ -30,6 +30,12 @@ releases:
       product: ocp
       stream: nightly
       version: "4.14"
+  source:
```

Comment:
openshift-extended-upgrade-pre is not referenced in openshift-upgrade-qe-test-e2e, so we do not need the setting here.

Reply:
remove it, thanks